src/css/styles.scss

```diff
@@ -127,16 +128,16 @@ main {
     font-size: 1.3em;
   }
 
-  .log-button {
-    background-color: #9dacea;
-    color: white;
-    padding: 10px;
-    border-radius: 5px;
-  }
-
-  .hide {
-    display: none;
-  }
+  // .log-button {
```

Comment:
@spaceplesiosaur Since we are not sure when we are going to be turning this in lets get rid of all the commented out code.  Not sure about you guys but I seem to forget to do this every time.

Reply:
Thanks, removed it! @ckoga